wikihow2zim/scraper.py

```diff
@@ -399,15 +427,13 @@ def build_filters_lists(self):
                     if not line or line.startswith("#"):
                         continue
                     if line.startswith("Category:"):
```

Comment:
Following block would fail if line is `Category:`. Using a regexp would work: `if re.match(r"Category:.+", line):`